<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tabReq">
       <attribute name="title">
        <string>Requirements</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
          </property>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_ReqIDFilter"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Search for requirements content:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_ReqContentFilter"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Search for requirements ID:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>250</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderDefaultSectionSize">
           <number>30</number>
          </attribute>
          <column>
           <property name="text">
            <string>Requirement ID</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Requirement Version</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Requirement Content</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tabErrors">
       <attribute name="title">
        <string>Errors</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableWidget" name="tableWidget_Errors">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderMinimumSectionSize">
           <number>150</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Requirement ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Error Description</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuProject">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Project Files</string>
     </property>
     <property name="icon">
      <iconset resource="qt_resource_file.qrc">
       <normaloff>:/actions/ico/16x16/actions/fileopen.png</normaloff>:/actions/ico/16x16/actions/fileopen.png</iconset>
     </property>
     <addaction name="action_AddFile"/>
     <addaction name="action_DelFile"/>
     <addaction name="action_OpenFile"/>
     <addaction name="separator"/>
    </widget>
    <widget class="QMenu" name="menuExport">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>Export</string>
     </property>
     <property name="icon">
      <iconset resource="qt_resource_file.qrc">
       <normaloff>:/actions/ico/16x16/actions/db_comit.png</normaloff>:/actions/ico/16x16/actions/db_comit.png</iconset>
     </property>
     <addaction name="action_ExportRequirementsAsExcelFile"/>
    </widget>
    <addaction name="action_NewProject"/>
    <addaction name="action_OpenProject"/>
    <addaction name="separator"/>
    <addaction name="action_SaveProject"/>
    <addaction name="action_SaveProjectAs"/>
    <addaction name="separator"/>
    <addaction name="menuProject"/>
    <addaction name="separator"/>
    <addaction name="menuExport"/>
    <addaction name="separator"/>
    <addaction name="action_Quit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="action_AboutPlatform"/>
    <addaction name="action_AboutReqExt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="locale">
     <locale language="English" country="UnitedStates"/>
    </property>
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="action_ExtractRequirements"/>
    <addaction name="separator"/>
    <addaction name="action_CheckForErrors"/>
    <addaction name="separator"/>
    <addaction name="action_Options"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget_ProjectFiles">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>200</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>800</width>
     <height>200</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Project Files</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="toolButton_AddFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Add file</string>
         </property>
         <property name="icon">
          <iconset resource="qt_resource_file.qrc">
           <normaloff>:/actions/ico/16x16/actions/edit_add.png</normaloff>:/actions/ico/16x16/actions/edit_add.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_DelFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Del file</string>
         </property>
         <property name="icon">
          <iconset resource="qt_resource_file.qrc">
           <normaloff>:/actions/ico/16x16/actions/edit_remove.png</normaloff>:/actions/ico/16x16/actions/edit_remove.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButton_OpenFile">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string>Open file</string>
         </property>
         <property name="icon">
          <iconset resource="qt_resource_file.qrc">
           <normaloff>:/actions/ico/16x16/actions/edit.png</normaloff>:/actions/ico/16x16/actions/edit.png</iconset>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>189</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>true</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>false</bool>
       </property>
       <property name="itemsExpandable">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <property name="allColumnsShowFocus">
        <bool>true</bool>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="headerHidden">
        <bool>false</bool>
       </property>
       <property name="expandsOnDoubleClick">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>400</number>
       </attribute>
       <attribute name="headerHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>400</number>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="headerStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">Files</string>
        </property>
        <property name="icon">
         <iconset resource="qt_resource_file.qrc">
          <normaloff>:/actions/ico/16x16/mimetypes/doc.png</normaloff>:/actions/ico/16x16/mimetypes/doc.png</iconset>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">Regular expression</string>
        </property>
        <property name="icon">
         <iconset resource="qt_resource_file.qrc">
          <normaloff>:/Pixmap/ico/16x16/mimetypes/applix.png</normaloff>:/Pixmap/ico/16x16/mimetypes/applix.png</iconset>
        </property>
       </column>
       <item>
        <property name="text">
         <string>Project</string>
        </property>
        <property name="icon">
         <iconset resource="qt_resource_file.qrc">
          <normaloff>:/Pixmap/ico/16x16/filesystems/folder_open.png</normaloff>:/Pixmap/ico/16x16/filesystems/folder_open.png</iconset>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsDragEnabled|ItemIsUserCheckable</set>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="action_AddFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/edit_add.png</normaloff>:/actions/ico/16x16/actions/edit_add.png</iconset>
   </property>
   <property name="text">
    <string>Add requirements file...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="action_Quit">
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/exit.png</normaloff>:/actions/ico/16x16/actions/exit.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="toolTip">
    <string>Exit application</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="action_AboutReqExt">
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/info.png</normaloff>:/actions/ico/16x16/actions/info.png</iconset>
   </property>
   <property name="text">
    <string>About Requirements Extractor</string>
   </property>
   <property name="statusTip">
    <string>Show Requirements Extractor details</string>
   </property>
  </action>
  <action name="action_AboutPlatform">
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/info.png</normaloff>:/actions/ico/16x16/actions/info.png</iconset>
   </property>
   <property name="text">
    <string>About Platform</string>
   </property>
   <property name="statusTip">
    <string>Show details about your platform</string>
   </property>
  </action>
  <action name="action_Options">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/configure.png</normaloff>:/actions/ico/16x16/actions/configure.png</iconset>
   </property>
   <property name="text">
    <string>Options</string>
   </property>
  </action>
  <action name="action_ExportRequirementsAsExcelFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/mimetypes/spreadsheet_document.png</normaloff>:/actions/ico/16x16/mimetypes/spreadsheet_document.png</iconset>
   </property>
   <property name="text">
    <string>All requirements in Excel file...</string>
   </property>
  </action>
  <action name="action_OpenProject">
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/project_open.png</normaloff>:/actions/ico/16x16/actions/project_open.png</iconset>
   </property>
   <property name="text">
    <string>Open project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_SaveProject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/filesave.png</normaloff>:/actions/ico/16x16/actions/filesave.png</iconset>
   </property>
   <property name="text">
    <string>Save project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="action_SaveProjectAs">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/filesaveas.png</normaloff>:/actions/ico/16x16/actions/filesaveas.png</iconset>
   </property>
   <property name="text">
    <string>Save project as...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="action_NewProject">
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/folder_new.png</normaloff>:/actions/ico/16x16/actions/folder_new.png</iconset>
   </property>
   <property name="text">
    <string>New Project...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="action_ExtractRequirements">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/run.png</normaloff>:/actions/ico/16x16/actions/run.png</iconset>
   </property>
   <property name="text">
    <string>Extract Requirements</string>
   </property>
  </action>
  <action name="action_CheckForErrors">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/run.png</normaloff>:/actions/ico/16x16/actions/run.png</iconset>
   </property>
   <property name="text">
    <string>Check for errors</string>
   </property>
  </action>
  <action name="action_ExtractFlows">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/run.png</normaloff>:/actions/ico/16x16/actions/run.png</iconset>
   </property>
   <property name="text">
    <string>Extract Flows</string>
   </property>
  </action>
  <action name="action_DelFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/edit_remove.png</normaloff>:/actions/ico/16x16/actions/edit_remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove selected file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionExport_2">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="action_OpenFile">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="qt_resource_file.qrc">
     <normaloff>:/actions/ico/16x16/actions/edit.png</normaloff>:/actions/ico/16x16/actions/edit.png</iconset>
   </property>
   <property name="text">
    <string>Open selected file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>toolButton_AddFile</tabstop>
  <tabstop>toolButton_DelFile</tabstop>
  <tabstop>toolButton_OpenFile</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineEdit_ReqIDFilter</tabstop>
  <tabstop>lineEdit_ReqContentFilter</tabstop>
  <tabstop>tableWidget_Errors</tabstop>
  <tabstop>tableWidget</tabstop>
 </tabstops>
 <resources>
  <include location="qt_resource_file.qrc"/>
 </resources>
 <connections/>
</ui>
